src/styles/styles.scss

```
@@ -2574,3 +2598,340 @@ ol {
+@media (prefers-reduced-motion: reduce) {
+  .theme-toggle,
+  .theme-toggle * {
+    transition: none !important;
+  }
+}
+
+/* =========================================================
+   Motion polish — global animations + micro-interactions
+   ========================================================= */
+
+@keyframes floatSubtle {
```

Stylelint (17.11.1)

[error] Expected keyframe name "floatSubtle" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)